pkg/cmdline/cmdline_linux.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package cmdline is parser for kernel command-line args from /proc/cmdline.
+//
+// It's conformant with
+// https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html,
+// though making 'var_name' and 'var-name' equivalent may need to be done
+// separately.
+package cmdline
+
+import (
+	"os"
+)
+
+const cmdLinePath = "/proc/cmdline"
+
+var procCmdLine *CmdLine
+
+func cmdLine(n string) *CmdLine {
```

Comment:
PR description says "Only use once.Do in one function, not everywhere." but I can't find it in the PR.   Was this mistakenly left out and belonging here?

Reply:
once.Do is gone. It was not even needed.